api/descriptions/defenders/serverless-bundle_post.md

```diff
@@ -1,5 +1,11 @@
 Downloads a ZIP file with serverless Defender bundle.
 
+Refer to the following values to assign:
+* For `provider`: Use `aws` or `azure`.
+* For `runtime`: 
+    * If you use `aws`, then you can add the runtime values: `python`, `python3.6`, `python3.7`, `python3.8`, `python3.9`, `nodejs12.x`, `nodejs14.x`, `dotnet6`, `java8`, `java11`, or `ruby2.7`.
+    * If you use `azure`, then you can add `dotnet6`.
```

Comment:
dotnet6, dotnet3

Reply:
For both aws and azure or only azure?

Reply:
Only azure.

Reply:
Thanks, I corrected. Would you be able to approve this PR?